### PR TITLE
chore(stringify): stringify otherData in applicationError

### PR DESCRIPTION
**Pull request by JamesSingleton**
This is the reasoning for this change: https://github.com/americanexpress/one-app/issues/36#issuecomment-587152336

### CODE REVIEW

**Comment by 10xLaCroixDrinker** [comment]
this is a feature, not a chore

**Comment by PixnBits** [comment]
> Larger Strategy Question: What about building `JSON.stringify` into `addErrorToReport`?

[Already present](https://github.com/americanexpress/one-app-ducks/blob/v4.0.0/src/errorReporting.js#L120).

Is the issue where [the `[Object object]`](https://github.com/americanexpress/one-app/issues/36#issuecomment-587279228) is coming from? That looks like the "location" value might have had a `toJSON` method that returned an object instead of a string? (coercion somewhere)

_Might_ be related to [`queue` being de-immutabilized](https://github.com/americanexpress/one-app-ducks/blob/v4.0.0/src/errorReporting.js#L110) (😅) after being [immutabilized](https://github.com/americanexpress/one-app-ducks/blob/v4.0.0/src/errorReporting.js#L60)?

File: src/error.js [hunk 1]
```diff
@@ -35,11 +35,11 @@ export const clearError = () => ({
 });
 
 export const applicationError = (code, error = new Error('application error'), otherData = {}) => (dispatch) => {
-  dispatch(addErrorToReport(error, {
+  dispatch(addErrorToReport(error, JSON.stringify({
```
Comment (JAdshead) [inline src/error.js:38]: what about doing this in [formatErrorReport](https://github.com/americanexpress/one-app-ducks/blob/master/src/errorReporting.js#L38)

**Comment by infoxicator** [comment]
> > Larger Strategy Question: What about building `JSON.stringify` into `addErrorToReport`?
> 
> [Already present](https://github.com/americanexpress/one-app-ducks/blob/v4.0.0/src/errorReporting.js#L120).
> 
> Is the issue where [the `[Object object]`](https://github.com/americanexpress/one-app/issues/36#issuecomment-587279228) is coming from? That looks like the "location" value might have had a `toJSON` method that returned an object instead of a string? (coercion somewhere) _Might_ be related to [`queue` being de-immutabilized](https://github.com/americanexpress/one-app-ducks/blob/v4.0.0/src/errorReporting.js#L110) (😅) after being [immutabilized](https://github.com/americanexpress/one-app-ducks/blob/v4.0.0/src/errorReporting.js#L60)?
> 
> Double serializing/encoding data in JSON seems like a workaround, rather than a fix.

@PixnBits for posterity it wasn't immutable causing this, nodejs console will truncate objects by default so raised a fix with this https://nodejs.org/api/util.html#util_util_inspect_object_options
https://github.com/americanexpress/one-app-ducks/pull/20

**Pull request merged**